WireDomain/Sources/WireDomain/Event Decryption/UpdateEventDecryptor.swift

```diff
@@ -108,7 +108,7 @@ struct UpdateEventDecryptor: UpdateEventDecryptorProtocol {
         error: ProteusError
     ) async {
         // Do not notify the user if the error is just "duplicated".
-        if error == .outdatedMessage || error == .duplicateMessage {
```

Comment:
Not sure why we were also checking for outdated messages here, but I removed it since it's no longer exposed as an error.